### ISSUE TITLE
Making comment only for two levels 

### ISSUE THREAD

**Issue by priyank21** [risul3/laravel-like-comment#10]
Thanks for this great module. is there is any way to make comments only for two levels.

**Comment by risul3** [comment]
You are welcome. Sorry there is no easy way to do that.

**Comment by priyank21** [comment]
what if we limit the loop which is responsible for coments

**Comment by risul3** [comment]
Yes we can do that, but you also have to block commenting after two level. For example someone just commented on level two, and ajax is done, now you have to disable comment form. Checking level from JS  will take quite good amount of coding.

**Comment by priyank21** [comment]
thanks for giving me your valuable time.  i really appreciate it. i am going to modify it for two levels only can you give me some tips. like where to start what logic you will prefer if you are implementing it.

**Comment by yaromand** [comment]
thanks for giving me your valuable time. i really appreciate it. i am going to modify it for two levels only can you give me some tips. like where to start what logic you will prefer if you are implementing it.
